packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -127,131 +127,136 @@
   {
     "chunkShortName": "link-tile-model-signature",
     "size": 6275,
-    "gzipSize": 2710
+    "gzipSize": 2712
   },
   {
     "chunkShortName": "main",
-    "size": 71115,
-    "gzipSize": 23907
+    "size": 71445,
+    "gzipSize": 23986
   },
   {
     "chunkShortName": "marque",
-    "size": 5582,
-    "gzipSize": 2232
+    "size": 5584,
+    "gzipSize": 2233
   },
   {
     "chunkShortName": "modal",
     "size": 8519,
-    "gzipSize": 3676
+    "gzipSize": 3673
   },
   {
     "chunkShortName": "model-signature",
-    "size": 2964,
-    "gzipSize": 1459
+    "size": 2966,
+    "gzipSize": 1460
   },
   {
     "chunkShortName": "multi-select",
-    "size": 21202,
-    "gzipSize": 7052
+    "size": 21364,
```

Comment:
This got bigger by 150 bytes?

Reply:
Did not change anything at multiselect 🤷‍♀️